charts/emissary-ingress/values.yaml.in

```diff
@@ -406,38 +406,13 @@ prometheusExporter: # +doc-gen:break
 #   runAsUser: 8888
 
 
-# Configure the ambassador agent
-agent:
-  # If `true`, installs the ambassador-agent Deployment, ServiceAccount and ClusterRole in the ambassador namespace, enabling the Ambassador Cloud connectivity.
+ambassador-agent:
```

Comment:
Are we not able to keep this under the alias as well? 

If not we need to make sure we have this in the docs as an upgrade step since it is a breaking change.

Reply:
Played around with this a little and it seems like I can actually keep this under the alias. Fixed.